frontend/src/lib/apiClient.js

```diff
@@ -7,83 +7,98 @@ const baseURL = process.env.REACT_APP_API_URL || 'http://localhost:3000';
 
 const apiClient = axios.create({
   baseURL,
-  headers: { 'Content-Type': 'application/json' },
+  headers: { 'Content-Type': 'application/json' }
 });
 
 // AUTHENTICATION
 
-const createUser = data => apiClient.post('api/v1/auth', {
-  user: {
-    first_name: data.firstName,
-    last_name: data.lastName,
-    email: data.email,
-    password: data.password,
-  },
-});
+const createUser = data =>
+  apiClient.post('api/v1/auth', {
+    user: {
+      first_name: data.firstName,
+      last_name: data.lastName,
+      email: data.email,
+      password: data.password
+    }
+  });
 
-const loginUser = (data) => {
+const loginUser = data => {
   const credentials = { email: data.email, password: data.password };
   return apiClient.post('api/v1/auth/login', {
-    user: credentials,
+    user: credentials
   });
 };
 
 // CURRENT USER
 
-const getCurrentUser = () => apiClient.get('api/v1/auth', {
-  headers: {
-    Authorization: sessionStorage.jwt,
-  },
-});
+const getCurrentUser = () =>
+  apiClient.get('api/v1/auth', {
+    headers: {
+      Authorization: sessionStorage.jwt
+    }
+  });
 
 // USERS
 
-const getUsers = () => apiClient.get('api/v1/users')
-  .then(results => results.data);
+const getUsers = () => apiClient.get('api/v1/users').then(results => results.data);
 
-const getUser = userId => apiClient.get(`api/v1/users/${userId}`, {
-  headers: {
-    Authorization: sessionStorage.jwt,
-  },
-});
+const getUser = userId =>
+  apiClient.get(`api/v1/users/${userId}`, {
+    headers: {
+      Authorization: sessionStorage.jwt
+    }
+  });
 
-const suspendUser = (userId, isSuspended) => apiClient.patch(`api/v1/users/${userId}`, {
-  is_suspended: isSuspended,
-});
+const suspendUser = (userId, isSuspended) =>
+  apiClient.patch(`api/v1/users/${userId}`, {
+    is_suspended: isSuspended
+  });
 
-const adminEditUser = (data, userId) => apiClient.patch(`api/v1/users/${userId}`, {
-  first_name: data.firstName.value,
-  last_name: data.lastName.value,
-  email: data.email.value,
-});
+const adminEditUser = (data, userId) =>
+  apiClient.patch(`api/v1/users/${userId}`, {
+    first_name: data.firstName.value,
+    last_name: data.lastName.value,
+    email: data.email.value
+  });
 
 // RECIPES
 
-const getRecipes = (search) => {
+const getRecipes = search => {
   const query = search ? `?search=${search}` : '';
-  return apiClient.get(`api/v1/recipes${query}`)
-    .then(results => results.data);
+  return apiClient.get(`api/v1/recipes${query}`).then(results => results.data);
```

Comment:
I also think there's readability value to having the `.then` clause on a separate line from the axios call. This is almost certainly prettier "fixing" things that don't need to be fixed.